tests/holodeck.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: holodeck.nvidia.com/v1alpha1
+kind: Environment
+metadata:
+  name: HOLODECK_NAME
```

Comment:
Can we choose a different name here? Perhaps, something along the lines of gpu-operator-e2e-env ?

Reply:
I believe the holodeck action automatically generates a name, but I could be wrong. @ArangoGutierrez what is the recommendation for naming this?

Reply:
Since this is the same as for the device plugin, let's leave it as is for now -- pending guidance on what this should be.